### ISSUE TITLE
get the google snippet pointing to the right place

### ISSUE THREAD

**Issue by jpyuda** [usagov/vote-gov#23]
When users search something like "how do I register to vote?" on google, they get a specially-selected and visually distinct result (see screenshot below). Right now, it points to usa.gov/register-to-vote; we would like it to point to vote.gov.

![image](https://cloud.githubusercontent.com/assets/552095/15373663/be109ece-1d14-11e6-9340-f6a1e36e7e66.png)

(I will have more to add based on a thread with Google)

**Comment by jpyuda** [comment]
I reached out to Google to see about help with changing where this points. Their response what that is is algorithmic and they don't typically manually override what the algorithm comes up with. We could prevent the usa.gov page from appearing by using [the nosnippet tag](https://googleblog.blogspot.com/2007/02/robots-exclusion-protocol.html), but we'd have no guarantee that vote.gov would replace it.

I asked what would happen if we retired usa.gov/register-to-vote and put a 301 redirect pointing to vote.gov; my contact didn't know the answer, so we're waiting on a response from developer relations at Google.

Marking this as blocked until we hear back.
